schedule/yast/modify_existing_partition/modify_existing_partition@s390x.yaml

```diff
@@ -5,10 +5,15 @@ description: >
   create_hdd test suite.
 vars:
   YUI_REST_API: 1
+conditional_schedule:
```

Comment:
we don't use conditional schedule, we tried to avoid this solution a time ago, please just remove the test module for now.

Reply:
ok, I get it, I will submit a new PR, so I close this one, thanks.